dspace/config/submission-forms.xml

```diff
@@ -32,7 +32,7 @@
         <field>
           <dc-schema>authors</dc-schema>
           <dc-element>institution</dc-element>
-          <dc-qualifier>code</dc-qualifier>
+          <dc-qualifier>name</dc-qualifier>
```

Comment:
Watch out for this, it will break the auto-complete functionality for this field. Right now, we are using the Dilbert API to complete this with an institution code. We can modify the `UCLouvainAuthorAuthority` class file to use `authors.institution.name` field instead of `authors.institution.code` but the API does not provide a name, only a code.